react-components/src/utilities/FdmSDK.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type CogniteClient } from '@cognite/sdk';
+
+type InstanceType = 'node' | 'edge';
+
+export type Item = {
```

Comment:
`Item` type is not used outside this class

Reply:
It's not exposed outside the package, so should be fine.